frames.test.js

```diff
@@ -0,0 +1,49 @@
+const Frames = require("./frames.js");
+
+describe("Frame", () => {
+  let frame;
+  beforeEach(() => {
+    frame = new Frames();
+  });
+
+  it("adds rolls points to frame's total points", () => {
+    frame.createNewFrame([3, 4]);
+
+    expect(frame.totalPoints()).toEqual(7);
+  });
+
+  it("adds bonus points if previous frame was strike", () => {
+    frame.createNewFrame([10]);
+    frame.createNewFrame([3, 4]);
+
+    expect(frame.totalPoints()).toEqual(24);
+  });
```

Comment:
I am not sure if your code accommodates for a case of strike-->strike-->roll, in which case the roll will have to be added to both the previous frame, and the one before it as well.